### PR TITLE
Update gif image and readme - Collaborative Stage View 

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/tab-stage-view/csharp/README.md [hunk 1]
```diff
@@ -119,30 +119,38 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
...
 
-![Stage View](TabInStageView/Images/CollaborativeStageView.png)
+![Welcome Message](TabInStageView/Images/welcomeAction.png)
 
-- Opening stage view from Adaptive card via deep link.
+**Open the URL in tab stage view:**
+
+![InstallApp](TabInStageView/Images/ClickButton.png)
+
+**Opening Collaborative Stage View**. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/README.md:136]: `**Opening Collaborative Stage View**. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details.`

File: samples/tab-stage-view/csharp/README.md [hunk 2]
```diff
@@ -119,30 +119,38 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
...
+![LinkUnfurlingStageView](TabInStageView/Images/LinkUnfurlingStageView.png)
 
-![Link Unfurling Text](TabInStageView/Images/LinkUnfurlingText.png)
+**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
 
-![Link Unfurling Card](TabInStageView/Images/LinkUnfurlingCard.png)
+![LinkUnfurlingText](TabInStageView/Images/LinkUnfurlingText.png)
 
-![Link Unfurling StageView](TabInStageView/Images/LinkUnfurlingStageView.png)
+**Tab with execute deep link action to open stage view:**
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/csharp/README.md:150]: You are showing just button with deep-link but after clicking on it what will happen?

Please check all other screenshots and it's next steps like above.

File: samples/tab-stage-view/nodejs/Readme.md [hunk 3]
```diff
@@ -132,10 +132,18 @@ Also update `BaseUrl` according to your code runtime environment.
...
 
+**Install App:**
+
+![InstallApp](Images/Install.png)
+
 **Welcome message with feature explanation and Adaptive Card with actions:**
 
 ![Welcome Message](Images/welcomeAction.png)
 
+**Open the URL in tab stage view:**
```
Comment (ChetanSharma-msft) [inline samples/tab-stage-view/nodejs/Readme.md:143]: Please mention what will happen once you click on these buttons and add those screenshots as well.

Check all other places in Readme files.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Kindly run build pipeline before merging this PR.

File: samples/tab-stage-view/csharp/README.md [hunk 4]
```diff
@@ -119,30 +119,38 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
     - In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
 
 ## Running the sample
+- In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/csharp/README.md:122]: Fixed

File: samples/tab-stage-view/nodejs/Readme.md [hunk 5]
```diff
@@ -132,10 +132,18 @@ Also update `BaseUrl` according to your code runtime environment.
 ## Running the sample
 - In the navigation bar located at the far left in Teams, select the ellipses ●●● and choose your app from the list.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/nodejs/Readme.md:133]: Fixed

File: samples/tab-stage-view/nodejs/Readme.md [hunk 6]
```diff
@@ -144,13 +152,11 @@ Also update `BaseUrl` according to your code runtime environment.
 
 ![Stage View Deep Link](Images/viaDeeplink.png)
 
-**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
-
-![Link Unfurling Text](TabInStageView/Images/LinkUnfurlingText.png)
+![LinkUnfurlingStageView](Images/LinkUnfurlingStageView.png)
 
-![Link Unfurling Card](TabInStageView/Images/LinkUnfurlingCard.png)
+**Opening stage view from unfurling link. If you copy and paste a link from https://www.BotFramework.com into the compose message area the link will unfurl.**
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/nodejs/Readme.md:157]: Fixed

File: samples/tab-stage-view/csharp/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/csharp/README.md:136]: Fixed

File: samples/tab-stage-view/csharp/README.md (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/csharp/README.md:150]: Fixed

File: samples/tab-stage-view/nodejs/Readme.md (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-stage-view/nodejs/Readme.md:143]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -** 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/workflows/build-complete-samples.yml

image

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine. approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**